src/Http/Offer/CalendarValidationRequestBodyParser.php

```diff
@@ -37,6 +37,24 @@ public function parse(ServerRequestInterface $request): ServerRequestInterface
                 );
                 break;
 
+            case 'single':
+            case 'multiple':
+                if (isset($data->subEvent)) {
```

Comment:
I'd suggest to add `&& is_array($data->subEvent)` here. Because in the `foreach()` below the `$data->subEvent` is not guaranteed to be an array right now.